src/lib/update-alternatives/UI/alternatives_dialog.rb

```diff
@@ -22,50 +22,66 @@
 Yast.import "UI"
 
 module UpdateAlternatives
-
+  # List all alternatives for an alternative group.
   class AlternativesDialog < UI::Dialog
-
     MIN_WIDTH = 60
     MIN_HEIGHT = 20
 
+    def initialize
+      @mock_slaves = {
+        ed:  "<pre>editor.1.gz /usr/share/man/man1/ed.1.gz</pre>",
+        vim: "<pre>editor.1.gz /usr/share/man/man1/vim.1.
+editor.fr.1.gz /usr/share/man/fr/man1/vim.1.gz
+editor.it.1.gz /usr/share/man/it/man1/vim.1.gz
+editor.pl.1.gz /usr/share/man/pl/man1/vim.1.gz
+editor.ru.1.gz /usr/share/man/ru/man1/vim.1.gz
+</pre>"
+      }
+    end
+
     def dialog_content
       MinSize(
         MIN_WIDTH,
         MIN_HEIGHT,
         VBox(
           create_alternatives_table,
-          create_slaves_table,
-          HBox(
-          PushButton(Id(:set), _("Set alternative")),
-          PushButton(Id(:cancel), _("Cancel"))
-          )
+          RichText(Id(:slaves), _("Please select an alternative to view his slaves.")),
+          footer
         )
       )
     end
 
     def set_handler
-      finish_dialog()
+      finish_dialog
+    end
+
+    def auto_handler
+      log.info("You have use the \"Set automatic mode\" button")
```

Comment:
log is for developer and not for "you" so it should write something like `User selected "Set automatic mode" button`